aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb

```diff
@@ -0,0 +1,588 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8zMksa4C-uxX"
+   },
+   "source": [
+    "## Setup\n",
+    "\n",
+    "First, we need to ensure we have SageMaker Python SDK 1.x and Tensorflow 1.15.x. Then, import necessary Python packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "AcGt7hZX-uxY",
+    "outputId": "bc1c8c6e-f204-4d04-eef5-4abbf18024d6",
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -U --quiet --upgrade \"sagemaker\" \n",
+    "!pip install -U --quiet \"tensorflow==1.15.3\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tarfile\n",
+    "import numpy as np\n",
+    "import sagemaker\n",
+    "import time\n",
+    "from sagemaker.utils import name_from_base"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iWnz80a2-uxZ"
+   },
+   "source": [
+    "Next, we'll get the IAM execution role and a few other SageMaker specific variables from our notebook environment, so that SageMaker can access resources in your AWS account later in the example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "2O3slomA-uxa"
+   },
+   "outputs": [],
+   "source": [
+    "from sagemaker import get_execution_role\n",
+    "from sagemaker.session import Session\n",
+    "\n",
+    "role = get_execution_role()\n",
+    "sess = Session()\n",
+    "region = sess.boto_region_name\n",
+    "bucket = sess.default_bucket()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Ij91QgHSAH-m"
+   },
+   "source": [
+    "SageMaker [Neo supports Tensorflow 1.15.x](https://docs.amazonaws.cn/en_us/sagemaker/latest/dg/neo-supported-cloud.html). Check your version of Tensorflow to prevent downstream framework errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "HTMoVu8M_-M3",
+    "outputId": "22fbf21b-254d-45ca-a2c3-b693dc9ef68d"
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "print(tf.__version__)  # This notebook runs on TensorFlow 1.15.x or earlier"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "RSxGrG4j-uxa"
+   },
+   "source": [
+    "## Download U-Net Model\n",
+    "\n",
+    "The SageMaker Neo TensorFlow Serving Container works with any model stored in TensorFlow's [SavedModel format](https://www.tensorflow.org/guide/saved_model). This could be the output of your own training job or a model trained elsewhere. For this example, we will use a pre-trained version of U-Net model based on this [repo](https://github.com/kamalkraj/DATA-SCIENCE-BOWL-2018)."
```

Comment:
Nit: Missing "the" 
 "For this example, we will use a pre-trained version of the U-Net..."

Reply:
Thanks for reviewing it. I have addressed all the comments.